types/add-dom-event-listener/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for add-dom-event-listener 1.1
+// Project: https://github.com/yiminghe/add-dom-event-listener
+// Definitions by: janeluck <https://github.com/janeluck>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function addDomEventListener(target: HTMLElement | Document | Window, eventType: string, callback: (event: any) => any): addDomEventListener.Listener;
```

Comment:
It would be nice to properly type the `event` callback param, but I guess this is better than no types at all.